core/db_models/EEM_Form_Section.model.php

```diff
@@ -0,0 +1,306 @@
+<?php
+
+use EventEspresso\core\services\request\RequestInterface;
+
+/**
+ * Class EEM_Form_Section
+ * Model Fields:
+ *      FSC_ID int(10) unsigned NOT NULL AUTO_INCREMENT,
+ *      FSC_UUID binary(16) NOT NULL,
+ *      FSC_appliesTo varchar(255) NOT NULL,
+ *      FSC_belongsTo binary(16) NULL,
+ *      FSC_htmlClass text NULL,
+ *      FSC_order tinyint(2) unsigned NOT NULL DEFAULT 0,
+ *      FSC_relation tinytext NULL,
+ *      FSC_status tinytext NULL,
+ *      FSC_wpUser bigint(20) unsigned NULL,
+ *
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class EEM_Form_Section extends EEM_Base
+{
+    const APPLIES_TO_ALL         = 'all';
+
+    const APPLIES_TO_PRIMARY     = 'primary';
+
+    const APPLIES_TO_PURCHASER   = 'purchaser';
+
+    const APPLIES_TO_REGISTRANTS = 'registrants';
+
+    const STATUS_ACTIVE          = 'active';
+
+    const STATUS_ARCHIVED        = 'archived';
+
+    const STATUS_DEFAULT         = 'default';
+
+    const STATUS_TRASHED         = 'trashed';
```

Comment:
Archived and Trashed are better this way instead of the confusion in other models objects where they are the same.

Reply:
right... want to try and improve some of the old ways as we move forwards